src/content/blog/loan.mdx

```diff
@@ -52,45 +52,45 @@ Attenzione che alcune banche effettuano arrotondamenti al calcolo oppure impongo
 
-Strettamente legata al tasso è la durata del mutuo; la durata dovrebbe essere di norma la più corta facilmente sostenibile. Durata lunga significa rata più bassa ma interessi totali più alti ed un impegni finanziario più lungo, durata più corta invece significa meno interessi totali ed un impegni più corto ma una rata mensile più alta (quindi meno reddito a disposizione per il resto!). Trovate la giusta via di mezzo a meno che non ci siano ragioni particolari. Un mio facoltoso cliente ad esempio ha fatto fare al figlio mutuo a tasso fisso per 40 anni dicendomi “ma lo sa lei che fra 40 anni 500 euro al mese saranno niente?”; il ragionamento ci può stare, non lo condivido al 100%, ma lo accetto e l’ho assecondato. Detto questo, per concludere la sezione dedicata al tasso, ponderate attentamente e con calma la scelta del tasso (e della durata)! State per impegnarvi per i prossimo 10, 20, 30 anni o forse più a pagare ogni mese una somma! La scelta del tasso si porta quindi dietro una forte componente di rischio e non solo di convenienza economica!
+Strettamente legata al tasso è la durata del mutuo; la durata dovrebbe essere di norma la più corta facilmente sostenibile. Durata lunga significa rata più bassa ma interessi totali più alti ed un impegno finanziario più lungo, durata più corta invece significa meno interessi totali ed un impegni più corto ma una rata mensile più alta (quindi meno reddito a disposizione per il resto!). Trovate la giusta via di mezzo a meno che non ci siano ragioni particolari. Un mio facoltoso cliente ad esempio ha fatto fare al figlio mutuo a tasso fisso per 40 anni dicendomi “ma lo sa lei che fra 40 anni 500 euro al mese saranno niente?”; il ragionamento ci può stare, non lo condivido al 100%, ma lo accetto e l’ho assecondato. Detto questo, per concludere la sezione dedicata al tasso, ponderate attentamente e con calma la scelta del tasso (e della durata)! State per impegnarvi per i prossimo 10, 20, 30 anni o forse più a pagare ogni mese una somma! La scelta del tasso si porta quindi dietro una forte componente di rischio e non solo di convenienza economica!
```

Comment:
In questo paragrafo la modifica è alla seconda riga nella parola "impegno"...non so perché non sia stata evidenziata la diff :sweat_smile: